### PR TITLE
Port over Noxary 268.2 from Rosakiin's Noxary Firmware

### PR DESCRIPTION
**Pull request by mechmerlin**
With this, we'll have all 268s accounted for in QMK, so people won't be confused anymore. Or will they? mwahahhahaa.

### CODE REVIEW

File: keyboards/noxary/268_2/n268_2.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "n268_2.h"
```
Comment (noroadsleft) [inline keyboards/noxary/268_2/n268_2.c]: Shouldn't this be `#include QMK_KEYBOARD_H`?

Comment (mechmerlin) [reply keyboards/noxary/268_2/n268_2.c]: We've only been changing that in `keymap.c` so far. We should change the `new_project.sh` script to reflect this if we want to do this too.

Comment (noroadsleft) [reply keyboards/noxary/268_2/n268_2.c]: Oops, you're right, I misread the filename.

File: keyboards/noxary/268_2/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,83 @@
+#include "n268_2.h"
```
Comment (drashna) [inline keyboards/noxary/268_2/keymaps/default/keymap.c]: Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/noxary/268_2/n268_2.c [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	return process_record_user(keycode, record);
+}
+
+void led_set_kb(uint8_t usb_led) {
+	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+	led_set_user(usb_led);
+}
+
+void led_set_user(uint8_t usb_led) {
```
Comment (drashna) [inline keyboards/noxary/268_2/n268_2.c]: This should be the kb version, or defined weakly:
```suggestion
__attribute__((weak))
void led_set_user(uint8_t usb_led) {
```

File: keyboards/noxary/268_2/n268_2.c [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	return process_record_user(keycode, record);
+}
+
+void led_set_kb(uint8_t usb_led) {
+	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+	led_set_user(usb_led);
+}
+
+__attribute__((weak))
```
Comment (drashna) [inline keyboards/noxary/268_2/n268_2.c]: I think this is the issue: 
```suggestion
__attribute__ ((weak))
```

File: keyboards/noxary/268_2/n268_2.c [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "n268_2.h"
```
Comment (drashna) [inline keyboards/noxary/268_2/n268_2.c]:
```suggestion
#include "268_2.h"
```

**Comment by drashna** [comment]
Thanks!

**Pull request merged**